myst_nbs/gaussian_processes/GP-Latent.myst.md

```diff
@@ -6,50 +6,41 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python (PyMC3 Dev)
+  display_name: pymc-dev
   language: python
-  name: pymc3-dev
+  name: pymc-dev
 ---
 
-# Latent Variable Implementation
+(gp_latent)=
+# Gaussian Processes: Latent Variable Implementation
 
-The `gp.Latent` class is a direct implementation of a GP.  It is called "Latent" because the underlying function values are treated as latent variables.  It has a `prior` method, and a `conditional` method.  Given a mean and covariance function, the function $f(x)$ is modeled as,
+:::{post} Sept 28, 2022
+:tags: gaussian processes, time series
+:category: reference, intermediate
+:author: Bill Engels
+:::
+
++++
+
+The {class}`gp.Latent` class is a direct implementation of a Gaussian process without approximation.  Given a mean and covariance function, we can place a prior on the function $f(x)$,
```

Comment:
The target should be `pymc.gp.Latent`. If you want the text to be `gp.Latent` you need to do `` {class}`gp.Latent <pymc.gp.Latent>` ``

Reply:
whoops, I missed that.  Thanks for the tip